pkg/sdk/testint/system_functions_integration_test.go

```diff
@@ -173,3 +173,19 @@ func TestInt_PipeForceResume(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, sdk.RunningPipeExecutionState, pipeExecutionState)
 }
+
+// TODO [SNOW-1650249]: add positive tests for bundle enablement (add SYSTEM$SHOW_ACTIVE_BEHAVIOR_CHANGE_BUNDLES() and use it to always pick a bundle that can be enabled/disabled)
```

Comment:
Should we add bcr checks in our checks before integration / acceptance tests so we will always start with the expected set of features enabled / disabled ? Not in this change though.

Reply:
Yup, it has some challenges but added to the issue.